_format/1.1/index.md

```diff
@@ -585,6 +594,100 @@ that includes a [profile alias][profile aliases]:
+    a removal of all or part of the link's context.
+  * `relate`: The link's target points to a resource where a user agent can cause
+    a direct or indirect relationship to be established with the link's context.
+
+> Note: These link relation types correspond to operations described by the
+JSON:API specification and it is expected that implementations will use them to
+reference resources that will process requests according to [the requirements for
+creating, updating or deleting resources](https://jsonapi.org/format/1.1/#crud). However, this is not a strict
+limitation on their use.
+
+In the example below, a `self` link is a member of the [top-level][top level] links
+object that represents a collection of articles. In this case, the `rel` member
+of the link indicates that the client can add an article to this collection.
+
+```json
+"links": {
+  "self": {
+    "href": "http://example.com/articles",
+    "rel": ["self", "add"]
+  }
+}
+```
+
+In the next example, an `add` link is a member of a `comments` [relationship object][relationships]
+and that relationship object is a member of an `article`'s relationships object.
+In this case, the `rel` member of the link indicates that the client can create
+a new comment at the target URL and that the server will automatically associate
+the new comment with the article's `comments` field.
+
+```json
+"links": {
+  "self": "http://example.com/articles/1/relationships/comments",
+  "related": "http://example.com/articles/1/comments",
+  "add": {
+    "href": "http://example.com/articles/1/comments",
+    "rel": ["add", "relate"]
+  }
+}
+```
+
+In this example, a `self` link is a member of an `article`'s links object. In
+this case, the `rel` member of the link indicates that the client can edit, but
+not delete, the context article (perhaps another request with different
+authentication credentials would receive a link with a `remove` link relation
+type).
+
+```json
+"links": {
+  "self": {
+    "href": "http://example.com/articles/1",
+    "rel": ["self", "update"]
+  }
+}
+```
+
+#### <a href="#document-links-link-parameter-object" id="document-links-link-parameter-object" class="headerlink"></a> Link parameter objects
+
+"Link parameter objects" appear in link objects to represent a link's target
+attributes.
+
+A link parameter object **MAY** contain any of the following members: `hreflang`,
+`media`, `title`, and `type`. The values of these members **MUST** conform to their
+meanings as defined by [RFC8288 Section 3.4.1](https://tools.ietf.org/html/rfc8288#section-3.4.1).
+
+A link parameter object **MAY** contain other members as target attributes. Any
+target attribute that is in common usage in other link serialisations **SHOULD NOT**
+be used in a manner contrary to the commonly understood meaning of those
+attributes.
+
+Any additional member names **MUST** be valid target attribute names as defined by
+[RFC8288 Section 2.2](https://tools.ietf.org/html/rfc8288#section-2.2).
```

Comment:
It's unclear to me how this paragraph interacts with the previous paragraph.

Reply:
I combined it with the previous one. Let me know it still feels disjointed.

Reply:
My reading of this is slightly different. I think this is trying to specify that:
- *all* target attributes must conform to RFC8288 section 2.2
- the `hreflang`, `media`, `title` and `type` target attributes have a superset of requirements: they must *also* conform to RFC8288 section 3.4.1 (in addition to conforming to RFC8288 section 2.2)

If my interpretation is correct, I agree that this should be made more clear.

Reply:
I significantly cleaned up the section on target attributes. Very curious to know what you think.